network-api/networkapi/templates/previews/libraries/rcc/detail_page.html

```diff
@@ -0,0 +1,11 @@
+{# This template is only used for CMS previews of RCCDetailPages. #}
+
+{% extends "pages/libraries/rcc/detail_page.html" %}
+
+{% block project_leads %}
+    {% for author in page.preview_related_authors %}
```

Comment:
Overriding the "related authors" section to use data returned from `preview_related_authors`, instead of `localize_authors` which is used on the live page template.